config/v1/types_build.go

```diff
@@ -97,6 +105,14 @@ type BuildOverrides struct {
 	// tolerations set on a build pod.
 	// +optional
 	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
+
+	// ForcePull specifies whether to always check for a newer version of the image
+	// in the registry (true) or allow the use of the local image (if it exists)
+	// even if it is not the latest version (false).
+	// Enabling forcePull is only relevant for tag references since a SHA reference
+	// can by definition only refer to a specific version of an image.
+	// +optional
```

Comment:
Clarifying this comment a bit, since `ImageStreamTags` get translated to SHA references under the hood.

```suggestion
	// forcePull specifies whether all builds should check for a newer version of its images
	// in the respective registries. Enabling forcePull is only relevant for builds which use `DockerImage` tag references.
	// Builds which use `ImageStreamTags` or direct SHA references by definition refer to a specific version of an image.
	// +optional
```